src/pageObject.js

```diff
@@ -1,19 +1,39 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return 'Conduit logo';
+  }
 };
 
 class PageObject {
-
+  constructor(url) {
+    this.header = new Header();
+    this.url = url;
+  }
+  clickOnLogo() {
+    return `Click on the ${this.header.logo}`;
+  }
 };
 
 class ArticlePage extends PageObject {
-
+  constructor(url, slug) {
+    super(url);
+    this.slug = slug;
+  }
+  get commentButton() {
+    return '[Publish comment] button';
+  }
+  clickOnCommentButton() {
+    return `Click on the ${this.commentButton}`;
+  }
+  assertPageOpened() {
+    return `The ${this.url} is opened`;
+  }
 };
 
 module.exports = {
```

Comment:
The base filename 'pageObject.js' does not match the name of its primary named or default export. Consider renaming the file to 'PageObject.js' to conform with best practices 📁